components/NewRealmWizard/components/steps/AddCouncilForm.tsx

```diff
@@ -52,12 +52,18 @@ export const AddCouncilSchema = {
     }),
 }
 
-export interface AddCouncil {
+export type AddCouncilV2 = {
```

Comment:
Here is a clear example of a discriminated union for versioning. I am telling the compiler that if `_programVersion === 3` then expect `councilYesVotePercentage` to be defined. The compiler does know how to do this.